examples/light-switch-app/silabs/include/LightSwitchMgr.h

```diff
@@ -39,20 +49,69 @@ class LightSwitchMgr
         chip::EventId event;
     };
 
+    static constexpr uint8_t stepSize = 1;
+    static constexpr DataModel::Nullable<uint16_t> transitionTime = 0;
+    static constexpr chip::BitMask<OptionsBitmap> optionsMask = 0;
+    static constexpr chip::BitMask<OptionsBitmap> optionsOverride = 0;
+
+    struct Timer
+    {
+        typedef void (*Callback)(Timer & timer);
+
+        Timer(uint32_t timeoutInMs, Callback callback, void * context);
+        ~Timer();
+
+        void Start();
+        void Stop();
+        void Timeout();
+
+        Callback mCallback = nullptr;
+        void * mContext    = nullptr;
+        bool mIsActive     = false;
+
+        osTimerId_t mHandler = nullptr;
+
+    private:
+        static void TimerCallback(void * timerCbArg);
+    };
+
+    LightSwitchMgr() = default;
```

Comment:
Since the LightSwitchMgr is a singleton, the constructor shouldn't be public

Reply:
Moved it to private